web/src/main/java/com/wilma/web/controller/web/portal/EducatorPortalController.java

```diff
@@ -29,31 +46,147 @@ public String dashboard(Model model) {
+                "repliesForPosts", forumService.getPostRepliesByCategory(category)));
+        return "/educator/forum/forum-thread";
+    }
+
+    @GetMapping("/delete-post/{postId}")
+    public RedirectView deleteForumPost(@PathVariable Integer postId, @RequestParam String category, Model model){
+        forumService.deleteById(postId);
+        model.addAllAttributes(Map.of(
+                "currentPage", "forum",
+                "menuElements", UserConfiguration.educatorMenuElements,
+                "categoryName", category,
+                "postsByCategory", forumService.getPostByCategoryName(category),
+                "repliesForPosts", forumService.getPostRepliesByCategory(category)));
+        return new RedirectView("/educator/forum-thread?category=" + category);
+    }
+    //endregion
+
+    //region Expressions of interest
+    @GetMapping("/expressions-of-interest")
+    public String expressionsOfInterest(Model model) {
+        model.addAllAttributes(Map.of(
+                "currentPage", "Expressions Of Interest",
+                "menuElements", UserConfiguration.educatorMenuElements,
+                "approvedPositions", List.of(
+                        new ExpressionOfInterest(1, new Partner("Microsoft", "Microsoft"), new Date(), new Date(), Period.of(0,0,1), "Brisbane", "A sample job", false, false),
+                        new ExpressionOfInterest(2, new Partner("Google", "Google"), new Date(), new Date(), Period.of(0,11,1), "Perth", "Another sample job", false, false),
+                        new ExpressionOfInterest(3, new Partner("Apple", "Apple"), new Date(), new Date(), Period.of(1,0,0), "Sydney", "A placement example", false, false),
+                        new ExpressionOfInterest(4, new Partner("Amazon", "Amazon"), new Date(), new Date(), Period.of(1,1,1), "Melbourne", "Slavery with extra steps", false, false)
+                ),
+                "pendingPositions", List.of(
+                        new ExpressionOfInterest(1, new Partner("Microsoft", "Microsoft"), new Date(), new Date(), Period.of(0,0,1), "Brisbane", "A sample job", false, false),
+                        new ExpressionOfInterest(2, new Partner("Google", "Google"), new Date(), new Date(), Period.of(0,11,1), "Perth", "Another sample job", false, false),
+                        new ExpressionOfInterest(3, new Partner("Apple", "Apple"), new Date(), new Date(), Period.of(1,0,0), "Sydney", "A placement example", false, false),
+                        new ExpressionOfInterest(4, new Partner("Amazon", "Amazon"), new Date(), new Date(), Period.of(1,1,1), "Melbourne", "Slavery with extra steps", false, false)
+                )
+        ));
+        return "/educator/expressions-of-interest";
+    }
+    //endregion
+
+    //region Profile
+    @GetMapping("/profile")
+    public String EducatorProfile(Model model) {
+        model.addAllAttributes(Map.of(
+                "currentPage", "Profile",
```

Comment:
Menu item not being selected
```suggestion
"currentPage", "profile",
```